blocks/partners/__labels/partners__labels.css

```diff
@@ -0,0 +1,26 @@
+.partners__labels {
+  padding: 40px 0;
+  margin: 0 -10px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: baseline;
+  row-gap: 20px;
+}
+
+@media screen and (min-width: 600px) {
+  .partners__labels {
+    margin: 0;
+    padding: 60px 0;
+    gap: 30px 50px;
+  }
+}
+
+@media screen and (min-width: 1114px) {
```

Comment:
It seems like there should be more labels from 1000px (10 instead of 8) as there appear some empty spaces in the second row.